init/hive/init.sh

```diff
@@ -5,6 +5,7 @@
 
 sed -i '$d' /usr/local/sbin/start.sh
 sed -i '$d' /usr/local/sbin/start.sh
+sed -i 's|hdfs://localhost:9000|hdfs://hive:9000|g' /usr/local/hive/conf/hive-site.xml
```

Comment:
spark sql create database will use `hive.metastore.warehouse.dir` in hive-site.xml, we should correct the hdfs address